all/entrypoint.sh

```diff
@@ -67,7 +67,7 @@ while test $# -gt 0; do
             shift
             ;;
         -i) shift
-            EXTRA_INCLUDES="-I$1"
+            EXTRA_INCLUDES="$EXTRA_INCLUDES -I$1"
```

Comment:
this will add an extra space for the first include, but it's the cleanest logic without an additional `if` statement IMO. open to a more complex implementation if anyone feels strongly about this